doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_pm_optimization.rst

```diff
@@ -0,0 +1,214 @@
+
+Unretained hardware classification
+**********************************
+
+Some power states in the nRF54H20 SoC result in powering off certain peripherals.
+The state of these peripherals is not retained by hardware and must be restored by software before the peripheral is activated.
+
+See the following sections for the lists of peripheral groups and the related software modules responsible for restoring the peripheral's state for each group.
+
+Peripherals in local domains
+============================
+
+All local domains include a common set of hardware modules.
+In addition to these, most local domains also contain domain-specific peripherals.
+
+Common peripherals for all local domains
+----------------------------------------
+
+Each local domain contains a set of peripherals that are classified consistently across all local domains.
+The following table summarizes the active peripherals that need handling when exiting the *Local Suspend to RAM* state.
+
++---------------+--------------------+--------------------+--------------------+--------------------------+
+|Type           | List of the        | Source of data to  | Time of restoration| Software module          |
+|               | peripherals        | restore            |                    | responsible for restoring|
++===============+====================+====================+====================+==========================+
+|Active         | * ``MVDMA``        | Device driver's    | Decided by the     | The device driver        |
+|peripherals    |                    | code and data      | driver             |                          |
++---------------+--------------------+--------------------+--------------------+--------------------------+
+
+Peripherals specific to the Application Domain
+----------------------------------------------
+
+There are no peripherals specific to the Application Domain.
+
+Peripherals specific to the Secure Domain
+-----------------------------------------
+
+The Secure Domain contains additional peripherals that require handling in the *Local Suspend to RAM* state.
+
++---------------+--------------------+--------------------+--------------------+--------------------------+
+|Type           | List of the        | Source of data to  | Time of restoration| Software module          |
+|               | peripherals        | restore            |                    | responsible for restoring|
++===============+====================+====================+====================+==========================+
+|Active         | * ``CRACEN``       | Device driver's    | Decided by the     | The device driver        |
+|peripherals    |                    | code and data      | driver             |                          |
++---------------+--------------------+--------------------+--------------------+--------------------------+
```

Comment:
We should probably remove it, considering it should be implemented in IronSideSE, so not really important for the application developer